pkg/store/bucket.go

```diff
@@ -213,6 +217,22 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Help: "Compressed size of postings stored into cache.",
 	})
 
+	m.postingsFetchDuration = promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+		Name:    "thanos_bucket_store_postings_fetch_duration_seconds",
+		Help:    "Time it takes to fetch postings to respond a query.",
```

Comment:
```suggestion
		Help:    "Time it takes to fetch postings to respond a query.",
```

Can we elaborate what fetch means? (it means fetch from object storage with cache on top)

Also do you think we can put those in one metric and have label `op` like we have for other metrics? (:

Reply:
Yes, of course. Good idea.